server/services/distance.ts

```diff
@@ -3,9 +3,36 @@ import { Location } from "../types/Location";
 
 export default class DistanceService {  
 
-    getDistanceBetweenLocations(loc1: Location, loc2: Location): number {
-        // Math.hypot returns the root of the squared values of its variables. So this is sqrt( (x2 - x1)^2 + (y2 - y1)^2 ), much shorter than the older version
-        return Math.hypot(loc2.x - loc1.x, loc2.y - loc1.y);
+    getDistanceBetweenLocations(loc1: Location, loc2: Location) {
+        return Math.sqrt(this.getDistanceSquaredBetweenLocations(loc1, loc2));
```

Comment:
Is this equivalent to the previous version? Any particular reason why this needed to be changed?